src/desktop/apps/fair_info/index.coffee

```diff
@@ -1,12 +1,12 @@
 express = require 'express'
 routes = require './routes'
-{ redirectFairRequests } = require('../fair/fairRedirection')
+{ fairRedirectionMiddleware } = require('../artsy-v2/apps/fair/fairRedirectionMiddleware')
 
 app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
-app.use '/:id/*', redirectFairRequests, routes.assignFair
```

Comment:
A couple questions:

- Is the `routes.assignFair` behavior still needed if we're querying the fair via our Metaphysics GraphQL layer?
- Do we still rely on these `/info/*` routes?

Reply:
This is a separate app that I haven't looked at yet.